.github/workflows/remove-stale-branches.yml

```
@@ -0,0 +1,26 @@
+name: remove-stale-branches
```

I prefer a normal sentence here because that's how we name other workflows too. See: https://github.com/infonl/dimpact-zaakafhandelcomponent/actions
```suggestion
name: Remove stale branches
```